tools/grafana/alerting/alerts.yml

```diff
@@ -6,10 +6,167 @@ groups:
+              hide: false
+              intervalMs: 1000
+              legendFormat: __auto
+              maxDataPoints: 43200
+              range: true
+              refId: total_errors
+          - refId: max_errors
+            queryType: ''
+            relativeTimeRange:
+              from: 0
+              to: 0
+            datasourceUid: '-100'
+            model:
+              conditions:
+                - evaluator:
+                    params:
+                      - 80
+                      - 0
+                    type: gt
+                  operator:
+                    type: and
+                  query:
+                    params:
+                      - total_errors
+                  reducer:
+                    params: []
+                    type: max
+                  type: query
+              datasource:
+                name: Expression
+                type: __expr__
+                uid: __expr__
+              expression: total_errors
+              hide: false
+              intervalMs: 1000
+              maxDataPoints: 43200
+              reducer: max
+              refId: max_errors
+              type: reduce
+          - refId: total_success
+            queryType: ''
+            relativeTimeRange:
+              from: 600
+              to: 0
+            datasourceUid: awx_alert
+            model:
+              datasource:
+                type: prometheus
+                uid: awx_alert
+              editorMode: code
+              expr: >-
+                max(delta(awx_instance_status_total{instance="awx1:8013",
+                status="successful"}[30m]))
+              hide: false
+              intervalMs: 1000
+              legendFormat: __auto
+              maxDataPoints: 43200
+              range: true
+              refId: total_success
+          - refId: max_success
+            queryType: ''
+            relativeTimeRange:
+              from: 0
+              to: 0
+            datasourceUid: '-100'
+            model:
+              conditions:
+                - evaluator:
+                    params:
+                      - 0
+                      - 0
+                    type: gt
+                  operator:
+                    type: and
+                  query:
+                    params:
+                      - total_success
+                  reducer:
+                    params: []
+                    type: max
+                  type: query
+              datasource:
+                name: Expression
+                type: __expr__
+                uid: __expr__
+              expression: total_success
+              hide: false
+              intervalMs: 1000
+              maxDataPoints: 43200
+              reducer: max
+              refId: max_success
+              type: reduce
+          - refId: compare
+            queryType: ''
+            relativeTimeRange:
+              from: 0
+              to: 0
+            datasourceUid: '-100'
```

Comment:
@shanemcd we figured out how to give prometheus data source a deterministic uid
This -100 uid is for data that doesn't reference a true "datasource", rather variables previously defined in the alert.